client/components/catalog/Card/Tags/index.vue

```diff
@@ -48,23 +52,27 @@ export default {
   },
   data: () => ({ showTagDialog: false }),
   computed: {
-    exceededTagLimit: ({ repository }) => repository.tags.length >= TAG_LIMIT
+    tagCount: vm => get(vm.repository, 'tags.length', TAG_LIMIT) - 1,
```

Comment:
Wrong name (inherited from before; since -1 and defaults to tag limit)

Reply:
I'm not sure what you're referring to? If it's the naming of `tagCount`, I can't figure out more appropriate name in this context 🤷‍♂️ . Any suggestions maybe? :)

Reply:
@borna223 
Based on how it is used I would default it to 0 instead of TAG_LIMIT and moved `-1` to the `maxTagNameLength` computed => `clamp(vm.tagCount - 1, 0, 2)`